.github/workflows/git-flow.yml

```
@@ -0,0 +1,179 @@
+    if: |
+      github.event_name == 'pull_request' && github.base_ref == 'main' &&
+      (startsWith(github.head_ref, 'release/v') || startsWith(github.head_ref, 'hotfix/v'))
+    steps:
+      - uses: actions/checkout@v4
+
+  tag-jira-release:
+    needs: [deploy-to-develop]
+    uses: VirdocsSoftware/github-actions/.github/actions/tag-tickets-with-release.yml@ar-CORE-1467-create-sharable-jobs
+    with:
+      target-branch: 'origin/main'
+      jira-domain: ${{ env.JIRA_DOMAIN }}
```

actionlint (1.7.3)

context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)